tests/experimental/starry/rotation_test.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np
+import pytest
+from jax import config
+from jaxoplanet._src.experimental.starry.rotation import R
+
+config.update("jax_enable_x64", True)
+
+
+@pytest.mark.parametrize("lmax", [10, 7, 5, 4, 3, 2, 1, 0])
+@pytest.mark.parametrize("u", [(1, 0, 0), (0, 1, 0), (0, 0, 1), (0.5, 0.1, 0)])
+def test_R(lmax, u):
+    pytest.importorskip("sympy")
+    theta = np.pi / 4
+    expected = np.array(R_symbolic(lmax, u, theta)).astype(float)
+    calc = R(lmax, u)(np.array([theta]))[0]
+    np.testing.assert_allclose(calc, expected, atol=5e-12)
+
+
+def R_symbolic(lmax, u, theta):
+    import sympy as sm
+    from sympy.functions.special.tensor_functions import KroneckerDelta
+
+    def Dmn(l, m, n, alpha, beta, gamma):
+        """Compute the (m, n) term of the Wigner D matrix."""
+        sumterm = 0
+        # Expression diverges when beta = 0
+        if beta == 0:
+            beta = 1e-16
+        for k in range(l + m + 1):
+            sumterm += (
+                (-1) ** k
+                * sm.cos(beta / 2) ** (2 * l + m - n - 2 * k)
+                * sm.sin(beta / 2) ** (-m + n + 2 * k)
+                / (
+                    sm.factorial(k)
+                    * sm.factorial(l + m - k)
+                    * sm.factorial(l - n - k)
+                    * sm.factorial(n - m + k)
+                )
+            )
+        return (
+            sumterm
+            * sm.exp(-sm.I * (alpha * n + gamma * m))
+            * (-1) ** (n + m)
+            * sm.sqrt(
+                sm.factorial(l - m)
+                * sm.factorial(l + m)
+                * sm.factorial(l - n)
+                * sm.factorial(l + n)
+            )
+        )
+
+    def D(l, alpha, beta, gamma):
+        res = sm.zeros(2 * l + 1, 2 * l + 1)
+        for m in range(-l, l + 1):
+            for n in range(-l, l + 1):
+                res[m + l, n + l] = Dmn(l, m, n, alpha, beta, gamma)
+        return res
+
+    def Umn(l, m, n):
+        """Compute the (m, n) term of the transformation matrix from complex to real Ylms."""
```

Comment:
This line is causing the pre-commit checks to fail because it's too long. Can you break it into two lines?